### ISSUE TITLE
Робоконечности и температура

### ISSUE THREAD

**Issue by Olt1771** [TauCetiStation/TauCetiClassic#1252]
#### Подробное описание проблемы

Робоконечности получают урон от перепадов температур

#### Что должно было произойти

От небольших перепадов теператур не должен был нанестись урон

#### Что произошло на самом деле

Берём риг шахтёра идём на шахтёрку, в вас стреляет базилиск, вы становитесь бесполезным и искритесь. Стрельнули с термал пушки, небольшое повышение температуры, вы опять искритесь как новогоняя ёлка

#### Как повторить

Всё выше перечисленное

**Comment by Dennok** [comment]
Да, вообще, думаю стоилобы по переделать механ робоконечностей и тп. но этож кодить надо.

**Comment by fukkatsumichanpewpew** [comment]
> При температуре ниже -50…-60 °С большинство металлов становятся хрупкими.
